RealmSwift.xcodeproj/xcshareddata/xcbaselines/554A51C73FAED0A637EBFEBF.xcbaseline/6890B8D9-CE81-403E-8EF6-B95A367D65B5.plist

```diff
@@ -16,22 +16,12 @@
 					<string>Local Baseline</string>
 				</dict>
 			</dict>
-			<key>testCommitWriteTransactionWithCrossThreadNotification()</key>
```

Comment:
Ok now next question: is it worth keeping performance tests if we don't measure their performance?